api/internal/utils/pid.go

```diff
@@ -51,7 +94,7 @@ func ReadPID(filepath string) (int, error) {
 	if err != nil {
 		return -1, err
 	}
-	pid, err := strconv.Atoi(string(data))
+	pid, err := strconv.Atoi(string(bytes.Trim(data, " \n")))
```

Comment:
Do we need a blank space before `\n` ?

Reply:
in go1.16 I was getting `invalid pid: strconv.Atoi: parsing "3041138\n": invalid syntax` parsing error ( newline error) without any newline at the end. 

However blank space is not required.